Assets/Scripts/Layers/Adapters/DataTypeAdapters/WFSImportAdapter.cs

```diff
@@ -0,0 +1,292 @@
+using System.IO;
+using System.Xml;
+using UnityEngine;
+using GeoJSON.Net.Feature;
+using Newtonsoft.Json;
+using System;
+using Netherlands3D.Web;
+using Netherlands3D.CartesianTiles;
+using System.Collections.Generic;
+using Netherlands3D.Twin.Layers;
+using Netherlands3D.Twin.UI.LayerInspector;
+
+namespace Netherlands3D.Twin
+{
+    [CreateAssetMenu(menuName = "Netherlands3D/Adapters/WFSImportAdapter", fileName = "WFSImportAdapter", order = 0)]
+    public class WFSImportAdapter : ScriptableObject, IDataTypeAdapter
+    {
+        [SerializeField] private Material visualizationMaterial;
+        [SerializeField] private LineRenderer3D lineRenderer3D;
+        [SerializeField] private BatchedMeshInstanceRenderer pointRenderer3D;
+
+        private string geoJsonOutputFormat = "";
+        private string wfsVersion = "";
+        private const string defaultFallbackVersion = "2.0.0"; // Default to 2.0.0 (released in 2010, compliant with ISO standards)
+
+        public bool Supports(LocalFile localFile)
+        {
+            var cachedDataPath = localFile.LocalFilePath;
+            var sourceUrl = localFile.SourceUrl;
+
+            Debug.Log("Checking source WFS url: " + sourceUrl);
+
+            // There are a lot of different WFS urls formats in the wild. This is a very basic check to see if it's a WFS service url.
+            var getCapabilitiesRequest = sourceUrl.ToLower().Contains("request=getcapabilities");
+            var getFeatureRequest = sourceUrl.ToLower().Contains("request=getfeature");
+
+            if(!getCapabilitiesRequest && !getFeatureRequest)
+            {
+                Debug.Log("<color=orange>WFS url does not contain a GetCapabilities or GetFeature request.</color>");
+                return false;
+            }
+
+
+            //Check if a GetFeature operation with GeoJSON as output format is supported
```

Comment:
I have my doubts whether this is wanted in the supports method as you determine whether this adapter is suitable. The name of the adapter suggests it is for all WFS URLs, so that our adapter's logic doesn't support certain things can probably better be delegated to the Execute. Because the adapter is or is not correct but whether we fully supports it may be a different matter. This can save quite a bit of performance possibly

Reply:
Maybe a rename of the adapter could help? I did intend this Support check to check if it is a WFS that we support, not to check if it is a WFS in general.

They way you describe it it would seem to me that you can get a scenario where an adapter Support check would succeed stopping the chain, and then the Execute could fail.

Now we would be able to for instance import a GetFeature GeoJSON from a WFS service we do not support as 'streaming' service, or import an image from a WMS service with a projection system we do not support, etc. etc.

If the double parse of the local xml would be a performance problem we could simply cache it after the Support check; but I didnt notice that to be the bottleneck.

Reply:
Making adapter name more explicit for now (include GeoJSON) so it is clear that this is a specific adapter. (Gml would get a seperate one)